src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,24 @@
+import { Popup } from "./Popup";
+class PopupWithConfirmation extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
+    super(popupSelector);
+    this._handleFormSubmit = handleFormSubmit;
+    this._formSelector = this._popupSelector.querySelector(".modal__container");
```

Comment:
`_formSelector ` 
`_popupSelector`

Word `selector` means a **string** that is used for searching elements in the DOM